Nim/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿Console.WriteLine("Welcome to Nim");
+int matches = 24;
+
+drawAgain:
+Console.WriteLine("Remove 1-3 matches");
+Console.WriteLine("User");
+//int draw = int.Parse(Console.ReadLine());
+if (!int.TryParse(Console.ReadLine(), out int draw))
+{
+    Console.WriteLine("You can't do that, choose 1-3 please");
+    goto drawAgain;
+}
+
+if (draw > 3 ||(draw <=0))
```

Comment:
Technically, the user can choose to draw more matches than there's left. How could this be improved? :)